### ISSUE TITLE
Double clicking an FBX file in the Asset Browser doesn't immediately open Scene Settings

### ISSUE THREAD

**Issue by AMZN-stankowi** [o3de/o3de#11488]
**Describe the bug**
Double clicking an FBX file in the Asset Browser used to immediately open the Scene Settings to the scene manifest for that FBX file. Now it presents a popup selection.

![image](https://user-images.githubusercontent.com/4838196/186221635-d32065eb-7a18-4988-9775-7c9c4ab049e0.png)

If you select "Open in Asset Editor" on this selection, it opens a tool where you can't actually do anything.

![image](https://user-images.githubusercontent.com/4838196/186221860-5b408996-bec4-4b76-954e-d813cf428072.png)

**Assets required**
Any FBX file

**Steps to reproduce**
1. Build and launch Editor
2. Double click on an FBX in the Asset Browser
3. Notice it doesn't immediately open the Scene Settings tool to the scene manifest for that FBX, and instead presents a popup
- ![image](https://user-images.githubusercontent.com/4838196/186221635-d32065eb-7a18-4988-9775-7c9c4ab049e0.png)
4. Click "Open in Asset Editor"
5. Notice it opens up a window where you can't interact or do anything.
- ![image](https://user-images.githubusercontent.com/4838196/186221860-5b408996-bec4-4b76-954e-d813cf428072.png)

**Expected behavior**
Double clicking an FBX opens Scene Settings and doesn't present an option to also open in Asset Editor.

**Actual behavior**
Double clicking an FBX shows a popup to select.

**Screenshots/Video**
See above

**Found in Branch**
Development at dc576766a459cda94b083a84ea018197f9d13ffb

**Desktop/Device (please complete the following information):**
N/A

**Additional context**
This came up in a UX review on something else, this was new and unexpected behavior.

**Comment by monroegm** [comment]
@lawsonamzn Is this related to any of the changes you made recently?

**Comment by lawsonamzn** [comment]
No.  My best guess is that that double click action does the default action if there's only one action but shows a popup if there's more than 1 action available.  Maybe someone added the "Open In Asset Editor" action to all entries?  It shouldn't be openable in asset editor.

**Comment by sphrose** [comment]
This is working in the latest development. Right clicking and selecting "Edit Settings" opens the asset in the fbx settings editor, double clicking goes straight into the editor.

**Comment by lawsonamzn** [comment]
I just tested this and its broken in the latest development. 

To be more specific, double clicking on a FBX does not go into the FBX settings, as its supposed to.   Instead it shows up a popup.

You are not supposed to be able to open FBX with the Asset Editor, the "open In asset editor" should not be a registered Opener for this type.  Whatever is causing the spurios "open in asset editor" for all types is the bug.

**Comment by lawsonamzn** [comment]
It may depend on what products the FBX produces, actualy.  Looks like the source code for this popup menu works in this manner

WHEN the user double clicks:
1. Ask the system for anything that can "open" this image
2. Collect all the "openers"
3. If there's just one "opener" then just open it (ie, the fbx settings)
4. If there's more than one "opener" show the popup.

The problem is that the logic for populating the "openers" has a default case, and that default case is that it includes all children (as in, products!), and any product which is a "generic asset" handler loaded by generic asset system, causes the appearance open in "asset editor" since thats the defualt editor for generic assets.

Problem: Azbuffers are generic assets.  Anything that outputs an azbuffer will cause this menu to appear.

**Comment by lawsonamzn** [comment]
To summarize here, the way this menu works is polling for any listeners to the AssetBrowserInteractionNotificationBus and populate itself with whatever is returned.  If only one result comes back, its the default result, and automatically occurs if you double click.  But if more than one result is returned, it pops up a menu with the options.

In the case of these assets, it turns out that there is a handler listening that will return "open in asset editor" for anything that has a **product** asset that is openable by the Generic Asset Handler, and scenefiles (FBX and others) almost always output such an asset.  

The fix would be to implement a more graceful opt-in for types that need the asset editor UI, type-by-type instead of the  blunt instrument that is "anything loaded by the generic asset handler".
